src/io/sampling.jl

```diff
@@ -43,6 +46,15 @@ function adjust_samples(spec::DataFrame, df::DataFrame)::DataFrame
     not_fogged = (df.fogging .== 0) .& (df.SRM .== 0)
     df[not_fogged, contains.(names(df), "shade_")] .= 0.0
 
+    # Normalize MCDA weights for fogging scenarios
+    df[.!(not_fogged) .& (df.guided .> 0), weights_fog_crit.fieldname] .= mcda_normalize(
+        df[.!(not_fogged) .& (df.guided .> 0), weights_fog_crit.fieldname],
+    )
```

Comment:
SRM is applied on a regional basis at the moment - shouldn't `not_fogged` just check if `fogging` == 0?

Reply:
good point!